lib/big_query.ex

```diff
@@ -1,22 +1,33 @@
 defmodule BigQuery do
 
-  defdelegate podcasts(), to: BigQuery.Podcasts, as: :list
-  defdelegate podcast(id), to: BigQuery.Podcasts, as: :show
+  alias BigQuery.Podcasts, as: Podcasts
+  alias BigQuery.Episodes, as: Episodes
+  alias BigQuery.Downloads, as: Downloads
+  alias BigQuery.Impressions, as: Impressions
 
-  defdelegate episodes(), to: BigQuery.Episodes, as: :list
-  defdelegate episode(guid), to: BigQuery.Episodes, as: :show
+  defmodule Interval do
```

Comment:
Instead of assigning `conn.assigns.from` / `conn.assigns.to` / etc separately, the Interval and Group plugs now assign these structs instead.  So instead of passing around all 3 of these fields as params, I can just pass around the single `Interval` struct.